examples/dualheaded_datautils/dataloaders.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import print_function
+import syft as sy
+import torch
+from torch.utils.data import Dataset
+from torch.utils.data import DataLoader
+from torch.utils.data._utils.collate import default_collate
+from typing import List, Tuple
+from uuid import UUID
+from uuid import uuid4
+from torch.utils.data import SequentialSampler, RandomSampler, BatchSampler
+
+import datasets
+
+
+"""I think this is not needed anymore"""
```

Comment:
Do you mean we don't need the partitioned dataloader?

Reply:
No, I mean that the default pytorch dataloader in PyTorch works, so we do not need a custom one (for how it is done now). See the notebook for an example.